packages/runtime-common/virtual-network.ts

```diff
@@ -250,9 +225,11 @@ async function buildRequest(url: string, originalRequest: Request) {
   // the new url and the body as a Uint8Array.
 
   let body = null;
-  if (originalRequest.body) {
-    body = await getContentOfReadableStream(originalRequest.clone().body);
+
+  if (['POST', 'PUT', 'PATCH'].includes(originalRequest.method)) {
```

Comment:
Previous condition (`if (originalRequest.body)`) didn't work in Firefox because this browser will say the body is undefined even when it's actually present:

FF:
<img width="268" alt="image" src="https://github.com/cardstack/boxel/assets/273660/f7310ca6-afb2-4568-b5f8-0937edc8f749">

Chrome:
<img width="269" alt="image" src="https://github.com/cardstack/boxel/assets/273660/6b5e65ba-6fb8-4ccb-9655-f03bf3655a5b">

Reply:
weird!